packages/admin/src/form/FinalFormMultiSelect.tsx

```diff
@@ -0,0 +1,114 @@
+import { createStyles, InputBase, InputBaseProps, MenuItemProps as MuiMenuItemProps, MenuList, WithStyles, withStyles } from "@material-ui/core";
+import * as React from "react";
+import { FieldRenderProps } from "react-final-form";
+
+interface MenuItemProps extends MuiMenuItemProps {
+    children?: React.ReactNode | ((selected: boolean) => React.ReactNode);
+}
+interface FinalFormMultiSelectProps extends FieldRenderProps<string, HTMLDivElement> {
+    children: React.ReactElement<MenuItemProps>[];
+    withSearch?: boolean;
```

Comment:
why "with" prefix? that is usually used for hoc or factory patterns

Reply:
is searchable as a propName better ? :)

Reply:
sounds good.